example/docs/introduction/migration.md

```diff
@@ -34,26 +34,25 @@ import 'vue-advanced-cropper/dist/style.css';
 
 ## Changed the markup of cropper
 
-1. Background is limited by boundaries now. To set the background of the entire cropper set background for the entire cropper.
+1. The background is limited to within a bounding box now.
```

Comment:
I can offer the following variant:
> 1. Background is limited by the cropper boundary now. If you set `background-color` in `background-class` it won't fill the entire cropper, it will fill only the area that fits in the cropper boundary. The entire cropper could be wider or taller than its boundary. To set the background for the entire cropper set `background-color` in `class` now.

Reply:
Sounds much more clear, I made a slight modification to it. Let me know what you think.

1. Background is limited by the cropper boundary now. If you set the ```background-color``` in the ```background-class``` it won't fill the entire cropper, it will fill only the area that fits in the cropper boundary. The entire cropper could be wider or taller than its boundary. To set the background for the entire cropper set the ```background-color``` in the ```class attribute``` now.

```style="height: 512px; width: 512px; background: #DDD;"```

Reply:
> style="height: 512px; width: 512px; background: #DDD;"

This part is not necessary I suppose.

> in the `class attribute` now.

I would replace it by: "in the `class` attribute now".

Reply:
Agreed, updating the request now.